### DISCUSSION TITLE
Not working on Brave Windows 11

### DISCUSSION THREAD

**Discussion by Mozoloa** [proginosko/LeechBlockNG#516] [Q&A] [answered]
I've been spending these past few days reinstalling windows and for some reason LB doesn't work anymore, the timer doesn't appear, when it worked on my previous install and works on other computers.

Here's my unique set 
![image](https://github.com/user-attachments/assets/514b88d3-c030-4617-b33d-9e0c9104597c)

I've removed and reinstalled LB to no avail, service worker says 
```
Error in event handler: TypeError: Cannot read properties of undefined (reading 'allowedHost')
    at handleTabCreated (chrome-extension://blaaajhemilngeeffpbfkdjjoefldkok/background.js:1669:54)
```

**Comment by proginosko** [comment, marked as answer]
I don't know why that error message is appearing, but the problem with your setup is that you need to remove "https://" from each line (see note under text box 🙂).

**Comment by Mozoloa** [comment]
How can I be so blind, I guess I got used to setting it up so much that I didn't read anymore. Sorry for wasting your time

**Reply by proginosko** [reply]
No worries! I should probably make LB strip off those protocol prefixes anyway.
